packages/app/src/elements/icon.ts

```diff
@@ -558,13 +558,25 @@ export class PlIcon extends LitElement {
                 content: "\\f2dc";
             }
 
-            :host([icon="shield-check"]) > div::before {
+            :host([icon="audit-pass"]) > div::before {
                 content: "\\f2f7";
             }
 
-            :host([icon="shield-xmark"]) > div::before {
+            :host([icon="audit-fail"]) > div::before {
```

Comment:
Icon names in this component are usually chosen by their meaning in the context in the app, as opposed to the actual thing they are showing.